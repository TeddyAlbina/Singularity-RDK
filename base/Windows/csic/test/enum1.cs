using System;
namespace N {
enum A {X=Y-1, Y=1, Z=X+2, W=B.X+3};
enum B {W=3, X};
public class Test {
	static A x = A.Z;
	public static void Main() {
		Console.WriteLine("{0}={1}", x, (int)x);
		Console.WriteLine("{0}={1}", A.W, (int)A.W);
	}
}
}

