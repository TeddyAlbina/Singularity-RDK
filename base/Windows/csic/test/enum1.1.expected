Z=2
W=7
